src/analysis/properties.rs

```diff
@@ -106,6 +118,7 @@ fn analyze_property(
         .min()
         .or(prop.version)
         .or(Some(env.config.min_cfg_version));
```

Comment:
Not sure if this is the problematic line, but there is an issue that the `min_cfg_version` of the current crate is used for external objects. Therefore I now get

```diff
     can_default: Option<bool>,
+    #[cfg(any(feature = "gtk_v1_0", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "gtk_v1_0")))]
```

in libhandy.

Reply:
Indeed. I will drop it